Hcal/include/Hcal/Event/VisiblesVetoResult.h

```diff
@@ -0,0 +1,85 @@
+/*
+ *@file VisiblesVetoResult.h
+ *@brief Class used to encapsulate the results obstained
+         from VisiblesVetoProcessor
+ *@author Tyler Horoho, University of Virginia
+ */
+
+#ifndef EVENT_VISIBLESVETORESULT_H_
+#define EVENT_VISIBLESVETORESULT_H_
+
+#include <iostream>
+
+//   ROOT   //
+#include <TObject.h>
+
+namespace ldmx {
+
+class VisiblesVetoResult {
+ public:
+  /** Constructor */
+  VisiblesVetoResult();
+
+  /** Destructor */
+  virtual ~VisiblesVetoResult();
+
+  void setVariables(int nLayersHit, double xStd, double yStd, double zStd,
```

Comment:
Please change the variable names to it matches the coding rules: https://ldmx-software.github.io/developing/coding-rules.html#i-naming-conventions